renderer/shaders/hlsl.glsl

```diff
@@ -43,22 +43,22 @@
 #endif
 
 $typedef float3 packed_float3;
-#define make_half float
-#define make_half2 float2
-#define make_half3 float3
-#define make_half4 float4
-
-$typedef $min16int short;
```

Comment:
We will need to figure out how to detect `min16int` support and feed that into the shader via `#define`, so we can keep using it where it's supported.